recipes/vapeplot/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "vapeplot" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 50cda617664f69873e459d82eeef72a43f7511a6397bc1b6411199645239a4f2
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
This is purely cosmetic to have this more standardised.